src/app/login-form/login-form-register.component.ts

```diff
@@ -0,0 +1,71 @@
+import {Component, Inject} from '@angular/core';
+import {MAT_DIALOG_DATA, MatDialogRef} from '@angular/material';
+import {User, UserService} from '../api/user.service';
+
+import {FormControl, Validators} from '@angular/forms';
+import { MatSnackBar } from '@angular/material';
+
+@Component({
+  selector: `app-login-form-register.component`,
+  templateUrl: 'login-form-register.component.html',
+  styleUrls: ['./login-form.component.scss'],
+})
+export class LoginFormRegisterComponent {
+  constructor(public dialogRef: MatDialogRef<LoginFormRegisterComponent>,
+              private userService: UserService,
+              public snackBar: MatSnackBar,
+              @Inject(MAT_DIALOG_DATA) public data: any) {
+  }
+
+  usernameRegisterText: string;
+  passwordRegisterText: string;
+  emailText: string;
+  users: User = {
```

Comment:
I can remove users because we've made it public (and I wasn't using the returned user data to do anything)

Don't I still need to keep the decleration for usernametext/passwordtext/emailtext? Otherwise when i call it in the if(!this.emailText) then it won't exist in the file. 

I could very well be missing an easier way to deal with that though.